modules/administration-guide/partials/proc_applying-the-gitlab-authorized-application-secret.adoc

```diff
@@ -0,0 +1,60 @@
+:_content-type: PROCEDURE
+:description: Applying the GitLab-authorized application Secret
+:keywords: applying-the-gitlab-oauth-app-secret, apply-the-gitlab-oauth-app-secret, apply-gitlab-oauth-app-secret, apply-secret, applying-secret, apply-a-secret, applying-a-secret
+:navtitle: Applying the GitLab-authorized application Secret
+// :page-aliases:
+
+[id="applying-the-gitlab-authorized-application-secret_{context}"]
+= Applying the GitLab-authorized application Secret
+
+Prepare and apply the GitLab-authorized application Secret as follows:
+
+.Prerequisites
+* The preceding procedure, Setting up the GitLab-authorized application, has been completed.
```

Comment:
```suggestion
* The preceding procedure, Setting up the GitLab-authorized OAuth 2.0 application, has been completed.
```

Reply:
I didn't touch the procedure heading when implementing your suggestion, so this line needn't be updated.